source/install/config-ssl-http2-nginx.rst

```diff
@@ -0,0 +1,95 @@
+.. _config-ssl-http2-nginx:
+
+Configuring NGINX with SSL and HTTP/2
+=====================================
+
+Using SSL gives greater security by ensuring that communications between Mattermost clients and the Mattermost server are encrypted. It also allows you to configure NGINX to use the HTTP/2 protocol.
+
+Although you can configure HTTP/2 without SSL, both Firefox and Chrome browsers support HTTP/2 on secure connections only.
+
+You can use any certificate that you want, but these instructions show you how to download and install certificates from *Let's Encrypt*.
```

Comment:
I converted the \*Let's Encrypt\* into a link to the Let's Encrypt website and added a bit of description to help those who do not know about Let's Encrypt. See commit https://github.com/JeffSchering/docs/commit/787e1d004100f8fb2fdb4a0c1d120a704ade5dfe